spec/xcpretty/parser_spec.rb

```diff
@@ -101,6 +101,11 @@ module XCPretty
       @parser.parse(SAMPLE_COMPILE_XIB)
     end
 
+    it "parses compiling storyboards" do
+      @formatter.should receive(:format_compile_storyboard).with("Main.storyboard", "sample/Main.storyboard")
```

Comment:
Line is too long. [109/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.